packages/desktop-client/src/components/Page.js

```diff
@@ -20,8 +23,37 @@ export function usePageType() {
 }
 
 function PageTitle({ name }) {
+  if (isMobile()) {
+    return (
+      <View
+        style={{
+          alignItems: 'center',
+          backgroundColor: colors.b2,
```

Comment:
Let’s keep this `colors.n10` for a contrast to the other screens. 
```suggestion
          backgroundColor: colors.n10,
```

Reply:
Scratch that. Shouldn’t be reading code on my phone. This is for all `Page`s. 
Hmm. I really like how they’re differentiated by color today. Maybe a Page can be passed a theme color?

Reply:
`n10` was used for Settings in the native app. It made me feel like I was in the unadorned back room when I visited Settings. 

I can appreciate a desire to keep it all uniform. I lean toward separate colors myself, partly because I like how the main purple has been used as an accent against other colors. I could see any one color everywhere feeling a bit…heavy.

Reply:
For reference:

<img width=390 src=https://user-images.githubusercontent.com/25517624/231330374-820f0b75-2c31-4aa2-9bcd-7f8db1344667.png>

Reply:
Updated the settings page header to match Jed's screenshot